### ISSUE TITLE
Correction on issue #65

### ISSUE THREAD

**Issue by johnworkman** [vincentbernat/snimpy#67]
My apologies. I wasn't setting contextname through the parameters on Manager(). 

I was setting the context like this:
m._session._session._contextname = 'vlan-713'

It turns out that setting contextname when instantiating Manager doesn't work. I think you want to make this change:
```diff
diff --git a/snimpy/snmp.py b/snimpy/snmp.py
index 8a94e56..bc08c03 100644
--- a/snimpy/snmp.py
+++ b/snimpy/snmp.py
@@ -132,12 +132,12 @@ class Session(object):
         self._none = none
         if version == 3:
             self._cmdgen = cmdgen.CommandGenerator()
-            self._contextname = None
+            self._contextname = contextname
         else:
             if not hasattr(self._tls, "cmdgen"):
                 self._tls.cmdgen = cmdgen.CommandGenerator()
             self._cmdgen = self._tls.cmdgen
-            self._contextname = contextname
+            self._contextname = None
         if version == 1 and none:
             raise ValueError("None-GET requests not compatible with SNMPv1")
```

**Comment by vincentbernat** [comment]
Oops, thanks for the correction! This is fixed in 4a2fb046e3f6.

**Comment by vincentbernat** [comment]
For some reason, I am unable to test properly this feature. Is it now working for you?
